### PR TITLE
feat: Add new request type for repo settings

### PR DESCRIPTION
**Pull request by bmtcril**
We've been getting many requests for settings changes to repositories, so discussed in retro setting up a new request type better suited to handling those. This is it!

### CODE REVIEW

**Review by sarina — APPROVED** [review]
The way I test these is putting them in a repo I own, and previewing the issue type.

**Comment by brian-smith-tcril** [comment]
I tried looking at the template here https://github.com/openedx/axim-engineering/blob/bmtcril/repo_settings_issue_template/.github/ISSUE_TEMPLATE/04-github-request---repo-settings.yml

and saw an error

image

File: .github/ISSUE_TEMPLATE/04-github-request---repo-settings.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        in your own repositories or forks in your personal account.
+        
+        If you are not the maintainer on the repository we may add them to the 
+        ticket to make sure the permissions make sense to the maintainers.
+  - type: input
+    id: repository
+    attributes:
+      label: Repository
+      description: Name of the repository you would like altered
+      placeholder: ""
```
Comment (brian-smith-tcril) [inline .github/ISSUE_TEMPLATE/04-github-request---repo-settings.yml:23]: I think this line may be causing the error I left a comment about, do we need to set a placeholder value here? Would it work if we just removed this line?

```suggestion
```

Comment (sarina) [reply .github/ISSUE_TEMPLATE/04-github-request---repo-settings.yml:23]: I think you just omit the `""`: https://github.com/openedx/axim-engineering/blob/main/.github/ISSUE_TEMPLATE/01-github-request---onboarding.yml?plain=1#L16

Comment (brian-smith-tcril) [reply .github/ISSUE_TEMPLATE/04-github-request---repo-settings.yml:23]: It looks like `placeholder:` (with nothing after it) might do the trick?

https://github.com/openedx/axim-engineering/blob/bf23b6371b719d374e04dc04f52cb9dc4c4c7356/.github/ISSUE_TEMPLATE/03-github-request---access.yml?plain=1#L19

Comment (bmtcril) [reply .github/ISSUE_TEMPLATE/04-github-request---repo-settings.yml:23]: Bah, of course it's the line my editor told was an error and insisted on fixing.

**Pull request merged**